### DISCUSSION TITLE
Shape library

### DISCUSSION THREAD

**Discussion by xzavrel** [flxzt/rnote#282] [Ideas]
Hi there, 
I'm currently in the process of tool exploration. Is there any possibility to define custom shapes (signs, schema parts etc?) library I can simply drag to the document?

**Comment by flxzt** [comment]
Hi, currently there is no such feature built-in unfortunately. It is possible to import SVG's though, so you could simply add a folder containing those shapes as a workspace and drag those in from the filebrowser.

**Comment by flxzt** [comment]
now a feature request in #620
